src/web/app/pages-student/student-course-details-page/student-course-details-page.component.html

```diff
@@ -111,60 +111,57 @@ <h4 class="font-weight-bold">Self</h4>
             <span>{{student.email}}</span>
           </div>
         </div>
-        <div class="form-group row">
-          <div class="col-4 text-right">
-            <label>Google ID</label>
-          </div>
-          <div class="col-8">
-            <span>{{student.googleId}}</span>
-          </div>
-        </div>
       </div>
     </div>
   </div>
 </div>
 
-<div class="section" *ngIf="teammateProfiles">
+<div class="section" *ngIf="teammateProfiles && teammateProfiles.length">
```

Comment:
You don't need this check `teammateProfiles`?

I hope you know what the `teammateProfiles` in the condition is used for.

Same for other places.

Reply:
Done! Yes I am aware. 

This was something I just reused from the current implementation. I have updated accordingly.